netlink-packet-route/src/rtnl/link/nlas/prop_list.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{
+    constants::*,
+    nlas::{Nla, NlaBuffer, NlasIterator},
+    parsers::parse_string,
+    traits::Parseable,
+    DecodeError,
+};
+
+use anyhow::Context;
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum PropList {
+    Unspec(Vec<u8>),
```

Comment:
The kernel never use this UNSPEC. No need to include it anymore.

And you need to include `Other(DefaultNla)` for future properties of `IFLA_PROP_LIST`.

Reply:
Ok done